src/screens/Login/LoginScreen.tsx

```diff
@@ -61,21 +61,36 @@ const LoginScreen: React.FC<LoginScreenProps> = ({ setCurrentUser }) => {
   useEffect(() => {
     if (!loading && !error && loginData) {
       const { login } = loginData;
-      (async () =>
+      async () => {
         // set access token
-        await SecureStore.setItem(SecureStoreEnum.TOKEN, login.accessToken))();
-      setCurrentUser(login.user.username);
+        await SecureStore.setItem(SecureStoreEnum.TOKEN, login.accessToken);
+        await SecureStore.setItem(SecureStoreEnum.USER_INFO, {
+          username: login.user.username,
+          id: login.user.id,
+        });
+      };
+
+      setCurrentUser({ username: login.user.username, id: login.user.id });
       navigateToHome();
     } else if (error) {
-      (async () =>
-        await SecureStore.deleteItem(SecureStoreEnum.TOKEN).catch())();
+      async () => {
```

Comment:
This doesn't get executed. It's just declaring an async function. It needs to be an immediately invoked function. It needs the wrapper brackets